crates/node/src/tee/remote_attestation.rs

```diff
@@ -80,3 +81,44 @@ pub async fn submit_remote_attestation(
         .await
         .context("failed to submit attestation after multiple retry attempts")?
 }
+
+/// Periodically generates and submits fresh attestations at regular intervals.
+///
+/// This function runs indefinitely, generating a fresh attestation every 10 minutes
+/// and submitting it to the blockchain. It's designed to be spawned as a background task.
+pub async fn periodic_attestation_submission<T: TransactionSender + Clone>(
+    tee_authority: TeeAuthority,
+    tx_sender: T,
+    tls_sdk_public_key: near_sdk::PublicKey,
+    tls_public_key: VerifyingKey,
```

Comment:
```suggestion
    tls_public_key: VerifyingKey,
```

You can create the SDK key version from the `VerifyingKey` type. No need to request both arguments from the caller, when the underlying key is the same with differing formats.

Reply:
Good point, not sure why I didn’t catch that myself. Anyway, fixed here: https://github.com/near/mpc/pull/1202/commits/f4497f34067290b5793d2169f19dce072bab9827. ✅